server-base.R

```diff
@@ -25,7 +25,7 @@ zcols <- c("darkslategrey", "yellow")
 # expect pct-data as a sibling of pct-shiny
 data_dir_root <- file.path(shiny_root, '..', 'pct-data')
 # packages required
-cran_pkgs <- c("shiny", "RColorBrewer", "httr", "rgdal", "rgeos", "leaflet", "DT", "shinyjs")
+cran_pkgs <- c("shiny", "RColorBrewer", "httr", "rgdal", "rgeos", "leaflet", "DT", "shinyjs", "sp", "dplyr")
```

Comment:
Out of interest why?  I was working without these for me.  Also this seems unrelated to #311...